compiler/src/dotty/tools/repl/Rendering.scala

```diff
@@ -76,11 +76,8 @@ private[repl] class Rendering(parentClassLoader: Option[ClassLoader] = None) {
     val dcl = d.symbol.showUser
 
     try {
-      val resultValue =
-        if (d.symbol.is(Flags.Lazy)) Some("<lazy>")
-        else valueOf(d.symbol)
-
-      resultValue.map(value => s"$dcl = $value")
+      if (d.symbol.is(Flags.Lazy)) Some(dcl)
+      else valueOf(d.symbol).map(value => s"$dcl = $value")
```

Comment:
:+1: on printing the flag to the left (in `ReplPrinter`). On this change, FWIW, Scala 2 also uses `<lazy>` here. Not sure anybody cares tho, so merging anyway.
```
scala> lazy val x = 1
x: Int = <lazy>
```